exercises/nth-prime/nth_prime.py

```diff
@@ -1,2 +1,2 @@
-def nth_prime():
+def nth_prime(n):
```

Comment:
`n` is too abigiuos name, we as programmers should avoid such names. Maybe `number` would be a better choice?